offline_pitch_extractor/crepe_extractor.py

```diff
@@ -28,12 +28,12 @@ def crepe_pitch(audio, sr):
             if idx != 0:
                 idxs.append(idx)
 
-    print("Detected frequency: ", frequency[idxs][0].round())
-
     # print("time: ", time[idxs], "frequency: ", frequency[idxs], "confidence: ", confidence[idxs])
     # print("time: ", time, "frequency: ", frequency, "confidence: ", confidence)
     # print(len(audio), "\n", len(time), "\n",  len(frequency), "\n",len(confidence),  "\n", len(activation))
-    return time, frequency, confidence, activation
+    #return time, frequency, confidence, activation
+    freq = frequency[idxs][0].round()
```

Comment:
potser hauriem de retornar la mean? perque si la primera esta mal maybe es shit

Reply:
ho he canviat. Faig pull i merge directament, que tira be!